lib/fs_utils/generate.js

```diff
@@ -247,6 +248,21 @@ const generate = (path, targets, config, optimizers) => {
     });
 };
 
+generate.writeFile = writeFile;
+
+generate.writeStatics = (toRemove, toWrite) => {
+  const removePromise = toRemove.reduce((promise, file) => {
+    file.dispose();
+    return promise.then(() => unlink(file.destinationPath));
```

Comment:
So that if `app/assets/index.jade` is deleted we *also* delete `public/index.html`